### PR TITLE
Elaborating on SetShardTabletControl

### PR DESCRIPTION
**Pull request by FancyFane**
Signed-off-by: FancyFane <fane@planetscale.com>

Changes made to v14 in draft mode; once validated we can work on replicating changes to other versions. 

- Added in links to relevant commands
- Added in additional examples as usage and behavior can be ambiguous
- Added note to run `RefreshStateByShard` after making modifications
- Updated terms to reflect terminology used in the topology record

### CODE REVIEW

File: content/en/docs/14.0/reference/programs/vtctl/shards.md [hunk 1]
```diff
@@ -113,12 +113,38 @@ SetShardIsPrimaryServing <keyspace/shard> <is_serving>
...
+Sets the shardTabletControls record for a shard and tablet type in the topology service. Only use this for an emergency fix or after a finished vertical split or after a corrupted MoveTables action. The [MigrateServedFrom](../keyspaces#migrateservedfrom) and [MigrateServedType](../keyspaces#migrateservedtype) commands set this field appropriately already. Always specify the `denied_tables` flag for MoveTables, but never for Reshard operations.<br><br>
```
Comment (Anders-PlanetScale) [inline content/en/docs/14.0/reference/programs/vtctl/shards.md]: Replace shardTabletCrontrols with ShardTabletControl

File: content/en/docs/14.0/reference/programs/vtctl/shards.md [hunk 2]
```diff
@@ -113,12 +113,38 @@ SetShardIsPrimaryServing <keyspace/shard> <is_serving>
...
+
+To set the queryServiceDisabled, set `disable_query_service` to true; to unset the queryServiceDisabled provide `denied_tables` with an empty tablet set. Useful to fix vReplication operations gone wrong.<br><br>
```
Comment (Anders-PlanetScale) [inline content/en/docs/14.0/reference/programs/vtctl/shards.md]: Maybe note that `disable_query_service=false` does not do anything, instead you need to provide denied_tables with an empty table set.

Also fix tablet set, to table set

**Review by Anders-PlanetScale — CHANGES_REQUESTED** [review]

File: content/en/docs/14.0/reference/programs/vtctl/shards.md (hunk 1 above)
Comment (FancyFane) [reply content/en/docs/14.0/reference/programs/vtctl/shards.md]: Taking a look at this we may want to add the "s" to the end everywhere `ShardTabletControl` appears. The reasoning is this is the way it's defined in the topology service:

```sh
vtctlclient SetShardTabletControl -- --disable_query_service=true commerce/0 primary
vtctlclient TopoCat -- --cell zone1 --decode_proto_json  /keyspaces/commerce/SrvKeyspace  | jq .
```

```json
...
        "shardTabletControls": [
          {
            "name": "0",
            "queryServiceDisabled": true
          }
        ]
```

Comment (Anders-PlanetScale) [reply content/en/docs/14.0/reference/programs/vtctl/shards.md]: OK, yeah then we need to make sure we use shardTabletControls when discussing the topo record, and `SetShardTabletControl` when discussing the vtctl command.

Comment (FancyFane) [reply content/en/docs/14.0/reference/programs/vtctl/shards.md]: I've added additional clarification and copy/pasted they keys names in the topology server for each of these changes. For additional clarity, I've added the specific topology path which gets updated with these commands.

File: content/en/docs/14.0/reference/programs/vtctl/shards.md (hunk 2 above)
Comment (FancyFane) [reply content/en/docs/14.0/reference/programs/vtctl/shards.md]: I've fixed tablet set to read table set; I've added a warning with the note about `disable-query_service=false`.

**Review by Anders-PlanetScale — APPROVED** [review]

**Comment by FancyFane** [comment]
- Removed double dash in v13, v12, and v11
- Corrected flags and topology variables in v12 and v11 which use outdated language

**Review by Anders-PlanetScale — APPROVED** [review]
LGTM

**Pull request merged**